packages/notification-managment-spa/package-lock.json

```diff
@@ -0,0 +1,15716 @@
+{
+  "name": "notification-management-spa",
```

Comment:
As @sayak-sarkar said please change the name of spa and the folder file names associated with it